alphacsc/tests/test_learn_d_z_multi.py

```diff
@@ -13,7 +12,20 @@
 
 @pytest.mark.parametrize('window', [False, True])
 @pytest.mark.parametrize('loss', ['l2', 'dtw', 'whitening'])
-@parametrize_solver_and_constraint
+@pytest.mark.parametrize('rank1, solver_d, uv_constraint',
+                         [
+                             (True, 'auto', 'auto'),
+                             (False, 'auto', 'auto'),
+                             (False, 'fista', 'auto'),
+                             (True, 'joint', 'auto'),
+                             (True, 'joint', 'joint'),
+                             (True, 'joint', 'separate'),
+                             (True, 'fista', 'auto'),
+                             (True, 'fista', 'joint'),
+                             (True, 'fista', 'separate'),
+                             (True, 'alternate_adaptive', 'separate')
+                         ]
+                         )
```

Comment:
Why not use the one defined in `conftest`?

Reply:
I modified it to exclude `auto` values. That way we can use it in more places and avoid unnecessary testing for `auto` values. 

On the other hand, I wanted to make sure that those `auto` values are correctly converted in `learn_d_z_multi`.

Reply:
Ok, thx for the explanation.